Tests/FluentAssertions.Specs/Formatting/FormatterSpecs.cs

```diff
@@ -458,11 +458,7 @@ public void When_formatting_single_it_should_have_max_precision()
         string result = Formatter.ToString(value);
 
         // Assert
-#if NETCOREAPP3_0_OR_GREATER
-        result.Should().Be("0.33333334F");
-#else
-        result.Should().Be("0.333333343F");
-#endif
+        result.Should().BeOneOf("0.33333334F", "0.333333343F");
```

Comment:
I feel uncomfortable when we don't know exactly what the expectation is.
Maybe we can argue that it is not relevant for FluentAssertion itself.